app/people/btech/page.js

```diff
@@ -68,16 +21,21 @@ const rows = [
   [17, 2020275, 'ADITYA GHOSH', 2020],
   [18, 2020276, 'ADITYA AHUJA', 2020],
   [19, 2020277, 'BHAGESH VERMA', 2020],
-  [20, 202315, 'DIVYAM SINGH', 2020],
+  [20, 2020315, 'DIVYAM SINGH', 2020],
 ];
 
-export default function BtechStudents() {
+export default async function BtechStudents() {
+  const spreadsheetId = '1cYPbswNr4ZvvQhzkfIOaM9Q75cSDP1CSl_JELvmA4iQ';
+  const range = 'B-Tech!A2:D1000';
```

Comment:
i mentioned on the group, you don't need to give cells in range, just give B-Tech and entire sheet will come, if we give ranges, when admins add data, it won't reflect.